app-config.production.yaml

```diff
@@ -1,8 +1,8 @@
 app:
   # Should be the same as backend.baseUrl when using the `app-backend` plugin
-  baseUrl: http://localhost:7000
+  baseUrl: http://localhost:7007
```

Comment:
You may not want to change the production configuration file -- so I figured I'd point this out!